tvb_documentation/tutorials/tutorial_0_GettingStarted.rst

```diff
@@ -4,187 +4,218 @@
 
-5. Select *Connectivity ZIP*.
+2. Select *Connectivity ZIP*.
 
-6. Select the file *connectivity\_regions\_96.zip* found at
-   *TVB\_Distribution/tvb\_data/*.
+3. From your downloaded tvb-data folder select the file *connectivity_96.zip* found at *tvb_data/connectivity/*.
 
-.. figure:: figures/LinkAndShare_Uploaders.png
+.. figure:: figures/ImportConnectivity_Uploaders.png
    :alt: Supported data formats.
    :scale: 30%
 
-7. Add a personalized tag to this newly created datatype (e.g.
-   *conn\_96*). 
 
-8. Save the changes.
+4. Add a personalized tag to this newly created datatype (e.g.
+   *conn\_96*).
 
-.. figure:: figures/LinkAndShare_TagDatatype.png
+5. Save the changes.
+
+.. figure:: figures/ImportConnectivity_TagDatatype.png
    :alt: Add a personalized tag.
    :scale: 50%
 
 ------------------------
 
 Link And Share
---------------
+---------------
 
 1. Select the connectivity you want to share.
 
 2. In the *metadata overlay*, go to the tab *Links*. You’ll see a list with all
-   your projects. 
+   your projects except the current one.
 
-3. Link this datatype (connectivity) with the project you’ll share (e.g.
-   *Session\_II\_ShareAndLink\_b*).
+3. Link this datatype (connectivity) with the project you’ll share.
 
 .. figure:: figures/LinkAndShare_LinksTab.png
    :alt: Links tab.
-   :scale: 50%
+   :scale: 30%
 
+4. Go to *Project > List of all projects*.
 
-4. Link the time-series from simulation *SimulationConnectivity192*
+5. Switch to the project you shared your datatype (connectivity) with.
 
+6. Then at *Project > Data Structure* you should see
+the datatype (connectivity) that you shared (the name is written in italics).
 
-.. figure:: figures/LinkAndShare_LinkedProject.png
-   :alt: Link a datatype to another project.
-   :scale: 50%
+.. figure:: figures/LinkAndShare_LinkedDatatype.png
+   :alt: You can see the datatype that you shared from NewProject.
+   :scale: 30%
 
-5. Go to *Project > List of all projects*
+7. Go to *Project > List of all projects*.
 
-6. Switch to *ShareAndLink\_b*.
+8. Switch to *ModelingEpilepsy*.
 
-7. Then from *Project > Basic properties* share this
-   project with the other user account.
+9. Then from Project > Basic properties share this project with the other user account.
 
-8. Logout from your account and login with the other user account.
+.. figure:: figures/LinkAndShare_LinkedProject.png
+   :alt: Click on the checkbox next to the user you want to share the project with
+   :scale: 30%
+
+10. Logout from your account and login with the other user.
+
+You should be able to see the *ModelingEpilepsy* project in your other account as well.
 
-You should be able to see the connectivity matrix (and other datatypes)
-you linked from project *Session\_II\_ShareAndLink\_a*.
 
 ------------------------
 
-Export and Read
+Export and Read Datatype
 ---------------
 
-#. Go to *Project > Data structure*
+1. Select *ModelingEpilepsy* as your current project.
```

Comment:
Could be helpful to show a screenshot with the button to click in order to select a project